### PR TITLE
Governance documentation

### PR DESCRIPTION
**Pull request by Dinonard**
Introduces governance related docs.

* `Learn` section has been expanded with governance sub-section
  * governance overview with links to relevant Polkadot documentation
  * in-depth view at governance actions & how they are done on-chain
* `Use` section has been expanded with on-chain governance section explaining how to use **Subsquare** platform

### CODE REVIEW

**Review by vedhavyas — APPROVED** [review]

File: docs/use/governance/subsquare_guide.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+
+---
+
+The `Target` is the account to which the votes are being delegated to.
+The `Balance` is the amount of native currency we're delegating. This is the amount that will be _locked_.
+:::important
+The action of delegating voting power **DOES NOT** transfer the ownership of the tokens. They are merely _locked_ for normal use, but remain in the ownership of the delegator.
+:::
+
+The `Conviction` allows us to amplify the voting power being transferred, at the expense of a longer _unlock_ period.
```
Comment (ipapandinas) [inline docs/use/governance/subsquare_guide.md:185]:
```suggestion
The `Conviction` setting determines how much your voting power is amplified in exchange for a longer token lock period.
```

File: docs/use/governance/subsquare_guide.md [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+
+---
+
+After submitting the proposal, it will be displayed on the treasury page.
+The proposal name & description can be edited.
+
+<Figure caption="Treasury - 3" src={require('/docs/use/img/19_Subsquare_Treasury/treasury_3.png').default } width="100%" />
+
+:::note
+The exact same steps can be reused for the community treasury proposals.
```
Comment (ipapandinas) [inline docs/use/governance/subsquare_guide.md:165]:
```suggestion
You can reuse the exact same steps for community treasury proposals.
```

File: docs/use/governance/subsquare_guide.md [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+---
+
+Proposals can be edited by clicking on the `Edit` button. Title and description can be changed, even  a graphica image can be uploaded.
+Users can discuss the proposal in the comments section.
+
+<Figure caption="Public Proposal - 7" src={require('/docs/use/img/18_Subsquare_public_proposal/public_proposal_6.png').default } width="100%" />
+
+---
+
+Clicking on the `Second` button will allow the user to second the proposal, increasing the chances of it being tabled and upgraded to a referendum.
```
Comment (ipapandinas) [inline docs/use/governance/subsquare_guide.md:135]:
```suggestion
Clicking the `Second` button to support the proposal, increase its chances of being tabled and upgraded to a referendum.
```

**Review by ipapandinas — COMMENTED** [review]
Massive work @Dinonard on detailing everything, kudos 👏
I'm suggesting some typo and rework fixes to improve it.

File: docs/use/governance/subsquare_guide.md [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+The account votes with 50 SBY, conviction set to **4**.
+This means that the total amount of votes for this account will be: **50 SBY x 4 + 300 SBY = 500 SBY**.
+
+<Figure caption="Voting - 6" src={require('/docs/use/img/21_Subsquare_voting/voting_6.png').default } width="100%" />
+
+---
+
+The new total turnout is 200 SBY - 50 SBY from the first `aye` vote, another 50 SBY from the second `aye` vote, and 100 SBY from the delegated vote.
+
+The total votes toward `aye` is 600 SBY in total - 100 SBY from the first `aye` vote, and another 500 SBY from the second `aye` vote.
```
[the remark this answers is not in the document]
Comment (Dinonard) [reply docs/use/governance/subsquare_guide.md:269]: I'll omit this since it might look like 600 SBY - 100 SBY 😅

**Review by Dinonard — COMMENTED** [review]
@ipapandinas a million thanks! 🙇 

Btw, feel free to push changes directly into my branch.

**Pull request merged**